src/network/networkctl.c

```diff
@@ -60,6 +60,21 @@ static bool arg_legend = true;
 static bool arg_all = false;
 static bool arg_stats = false;
 
+static int resolve_ifname(sd_netlink **rtnl, const char *name) {
+        int r;
+
+        r = parse_ifindex_or_ifname(name);
+        if (r > 0)
+                return r;
+        assert(r < 0);
+
+        r = rtnl_resolve_link_alternative_name(rtnl, name);
+        if (r > 0)
+                return r;
```

Comment:
hmm, shouldn't we check this already in parse_ifindex_or_ifname() itself?

i.e. introduce a if_nametoindex_harder() or so that is like if_nametoindex() but also checks alternative names? I mean, it's a pity the kernel doesn't support this natively in SIOCGIFINDEX but I think we should just add our own helper tha tdoes this anyway?

i.e. I am pretty sure we should just support alternative names everywhere, and not just in networkctl?